docs/prover/cli/options.md

```diff
@@ -156,6 +156,12 @@ the contract or the spec to rerun, we can just rerun on the `deposit` method:
 certoraRun --method 'deposit(uint)'
 ```
 
+If we discover a counterexample in several methods, we could rerun just those:
+
+```sh
+certoraRun --method 'deposit(uint)' --method 'transfer(address,uint256)'
```

Comment:
@naftali-g shouldn't this be either
```suggestion
certoraRun --method ['deposit(uint)', 'transfer(address,uint256)']
```

Reply:
No, the way to specify multiple methods from command line is with multiple usages of the `--method` flag. In the conf file it's with square brackets, like any list.

Reply:
Thanks for clarification!